Python/Tests/Core/DebugReplEvaluatorTests.cs

```diff
@@ -33,7 +33,7 @@
 
 namespace PythonToolsTests {
     [TestClass]
```

Comment:
I wonder whether we still need this "TestClass" attribute.

Reply:
Not needed. Will be removed in next commit.